core/src/test/java/com/github/mizool/core/validation/TestIdentifierValueAnnotation.java

```diff
@@ -51,57 +43,30 @@
         private List<String> identifiers;
     }
 
-    private Set<ConstraintViolation<TestIdentifierValueAnnotation.TestData>> runValidator(TestIdentifierValueAnnotation.TestData testData)
-    {
-        Validator validator = Validation.buildDefaultValidatorFactory().getValidator();
-        return validator.validate(testData);
-    }
-
-    private Set<ConstraintViolation<TestIdentifierValueAnnotation.TestListData>> runValidator(
-        TestIdentifierValueAnnotation.TestListData testData)
-    {
-        Validator validator = Validation.buildDefaultValidatorFactory().getValidator();
-        return validator.validate(testData);
-    }
-
     @Test
     public void testValidationOfUnacceptableValue()
     {
-        TestIdentifierValueAnnotation.TestData testData = new TestIdentifierValueAnnotation.TestData(EMPTY);
-        Set<ConstraintViolation<TestIdentifierValueAnnotation.TestData>> violations = runValidator(testData);
-        assertThat(violations).hasSize(1);
-        ConstraintViolation<TestIdentifierValueAnnotation.TestData> violation = Iterables.getFirst(violations, null);
-        String violatedAnnotation = violation.getConstraintDescriptor().getAnnotation().annotationType().getName();
-        assertThat(violatedAnnotation).isEqualTo(IdentifierValue.class.getName());
+        ValidatorAnnotationTests.assertUnacceptableValue(new TestIdentifierValueAnnotation.TestData(EMPTY),
```

Comment:
Remove `TestIdentifierValueAnnotation.`. `TestData` should suffice here and in the other test methods.